### DISCUSSION TITLE
Watris

### DISCUSSION THREAD

**Discussion by giscus** [aduros/wasm4#6] [Games]
# Watris

https://wasm4.org/play/watris?giscus=9936405df88419e8538749300emzP4%2BfFDIM%2Fqxk%2FKmtyyn3l4i5j8gAaLeRVzHXaYKjt%2BvWe5WvvjyGpTLhy%2FHeccHEXNHaEe3xpjQNHoLBrCfxBg7rBlkZYVKkajQNUhoADdmlgGSqeRct3oA%3D

**Comment by aduros** [comment]
This was a fun little game to hack together. Tetris is always the first game I build on any new framework :smile: I'm proud of the color scheme and palette cycling.

[Source code](https://github.com/aduros/wasm4/tree/main/examples/watris)

**Reply by EssaRightThere** [reply]
is nice. but they're having funny sound effects.
